src/main/java/seedu/duke/command/ListNoteCommand.java

```diff
@@ -23,22 +23,25 @@
             + PREFIX_DELIMITER + PREFIX_TAG + " TAG1...] "
             + "[up/down]";
 
-    private ArrayList<String> tags;
-    private boolean isSorted;
-    private Boolean isAscendingOrder;
-
+    /**
+     * Gets how the command is expected to be used.
+     *
+     * @return String representation of how the command is to be used.
+     */
     public static String getCommandUsage() {
         return COMMAND_USAGE;
     }
 
+    private ArrayList<String> tags;
+    private Boolean isAscendingOrder;
```

Comment:
Should we standardize using pointer Boolean or primitive boolean?

Reply:
For this, because the user can choose not to input the sort parameter, therefore the Boolean can hold a null value, which is used to check later

Reply:
@brandonywl Yup, so when I was using Boolean it was because there are 3 cases. Default (which is chronological order), ascending or descending. Hence I wanted to use Boolean which can hold true, false and null! boolean on the other hand cannot hold the null value.